presto-product-tests/README.md

```diff
@@ -43,6 +43,10 @@ broken.
 `hadoop-master`, `mysql`, `postgres`, `cassandra`, `presto-master`.
```

Comment:
That's kind of weird. First, not all these are needed. And any needed looks like a test scaffolding deficiency... but this is of topic I know